.github/workflows/pr.yml

```diff
@@ -41,6 +41,7 @@ jobs:
 
   algolia:
     uses: the-guild-org/shared-config/.github/workflows/algolia-integrity.yml@main
+    if: ${{ github.event.pull_request.head.repo.full_name == 'org/repo' }}
```

Comment:
@gilgardosh it needs to be changed to `== 'n1ru4l/envelop'` , no?

Reply:
```suggestion
    if: ${{ github.event.pull_request.head.repo.full_name == 'n1ru4l/envelop' }}
```

Reply:
oops 😅